project/scripts/bootstrapCmdTests

```diff
@@ -5,18 +5,18 @@ set -e
 source $(dirname $0)/cmdTestsCommon.inc.sh
 
 # check that benchmarks can run
-"$SBT" "scala3-bench/jmh:run 1 1 tests/pos/alias.scala"
-# The above is here as it relies on the bootstrapped library.
-"$SBT" "scala3-bench-bootstrapped/jmh:run 1 1 tests/pos/alias.scala"
-"$SBT" "scala3-bench-bootstrapped/jmh:run 1 1 -with-compiler compiler/src/dotty/tools/dotc/core/Types.scala"
+# "$SBT" "scala3-bench/jmh:run 1 1 tests/pos/alias.scala"
+# # The above is here as it relies on the bootstrapped library.
+# "$SBT" "scala3-bench-bootstrapped/jmh:run 1 1 tests/pos/alias.scala"
+# "$SBT" "scala3-bench-bootstrapped/jmh:run 1 1 -with-compiler compiler/src/dotty/tools/dotc/core/Types.scala"
 
-echo "testing scala.quoted.Expr.run from sbt scala"
-"$SBT" ";scala3-compiler-bootstrapped/scalac -with-compiler tests/run-staging/quote-run.scala; scala3-compiler-bootstrapped/scala -with-compiler Test" > "$tmp"
-grep -qe "val a: scala.Int = 3" "$tmp"
+# echo "testing scala.quoted.Expr.run from sbt scala"
+# "$SBT" ";scala3-compiler-bootstrapped/scalac -with-compiler tests/run-staging/quote-run.scala; scala3-compiler-bootstrapped/scala -with-compiler Test" > "$tmp"
+# grep -qe "val a: scala.Int = 3" "$tmp"
 
 
 # setup for `scalac`/`scala` script tests
-"$SBT" dist/pack
+# "$SBT" dist/pack
 
```

Comment:
Why did these tests get commented out?

Reply:
Leftover from local debugging (benchmarks, even such small takes a while to run)